### PR TITLE
Spanish translation strings

### PR DESCRIPTION
**Pull request by JoseDeFreitas**
_Refer to this issue: #77._

I've translated the files `strings.xml` and created folders with the name `values-es` that contain those files.
Note that, in order to follow Spanish grammar rules, I used characters such as "¡ ¿ á é í ó ú Á É Í Ó Ú".
I translated the strings according to how it should show up on the application.
Wanting to have this PR counted for the Hacktoberfest 2020. Tell me if any change should be made.
✌

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,715 @@
...
+    <string name="dark_theme_invite_a_friend_title">Regale a alguien el tema oscuro</string>
+    <string name="dark_theme_invite_sent">Su regalo ha sido enviado</string>
+    <string name="dark_theme_invites">Todo es mejor con amigos</string>
+    <string name="dark_theme_invites_action">Invite a alguien al lado oscuro</string>
+    <string name="dark_theme_invites_desc">Completar más de una acción te dará invitaciones. Haz click en el botón de abajo para desbloquear el tema oscuro para alguien</string>
+    <string name="dark_theme_no_more_invites">El servidor dijo que no tiene invitaciones. El servidor nunca se equivoca.</string>
+    <string name="dark_theme_send_instructions">Dime el nombre de usuario SAGRES de la persona que recibirá el regalo. Esta persona necesita estar conectada al UNESverso</string>
+    <string name="dark_theme_send_label">¿A quién deseas darle un regalo?</string>
+    <string name="dark_theme_send_warning">Si escribes el nombre incorrecto de la persona o si la persona ya tiene el tema oscuro, tu invitación puede ser consumida de todas formas :). También, puede pasar que la persona no recibe el regalo porque no está conectada :)string>
```
Comment (ForceTower) [inline app/src/main/res/values-es/strings.xml:264]: This line should end as `</string>`

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,715 @@
...
+    <string name="dynamic_feature_module_unavailable">Módulo no disponible</string>
+    <string name="eval_loading_evaluation">Cargando información</string>
+    <string name="evaluate_discipline">Evaluar especialidad</string>
+    <string name="evaluate_teacher">Evaluar profesor</string>
+    <string name="evaluated_teacher_image">Imagen del profesor</string>
+    <string name="evaluation_approvals">aprobaciones</string>
+    <string name="evaluation_completed_description">¡Gracias por evaluar! Tus respuestas ayudarán a otras personas en el futuro</string>
+    <string name="evaluation_completed_label">¡Evaluación completa!</string>
+    <string name="evaluation_create_knowledge_database">Creando una base de datos de conocimiento</string>
+    <string name="evaluation_create_knowledge_warning">Esto puede tardar un rato. Si dejas esta pantalla, puede reiniciarse :)string>
```
Comment (ForceTower) [inline app/src/main/res/values-es/strings.xml:353]: This one too `</string>`

**Review by ForceTower — CHANGES_REQUESTED** [review]
Really thanks for your contribution!

I highlighted 2 lines that didn't end correctly. That aside, big thanks for translating the app 👍

**Comment by JoseDeFreitas** [comment]
Hello! I've just fixed those issues. Thank you for catching them. I also reviewed the other `<string>` tags, I think everything is okay now.

**Review by ForceTower — COMMENTED** [review]
Nice!

**Review by ForceTower — APPROVED** [review]

**Pull request merged**